app/controllers/v2/reports_controller.rb

```diff
@@ -7,18 +7,21 @@ def index
       render_json reports
     end
     permission_for_action :index, Rbac::REPORT_READ
+    v2_permission_for_action :index, KesselRbac::REPORT_VIEW
 
     def show
       render_json report
     end
     permission_for_action :show, Rbac::REPORT_READ
+    v2_permission_for_action :show, KesselRbac::REPORT_VIEW
 
     def destroy
       report.delete_associated
       audit_success("Removed report #{report.id}")
       render_json report, status: :accepted
     end
     permission_for_action :destroy, Rbac::POLICY_DELETE
+    v2_permission_for_action :destroy, KesselRbac::REPORT_REMOVE
```

Comment:
Selfnote: the permission has been changed from `compliance:policy:delete` to `compliance:report:delete`.